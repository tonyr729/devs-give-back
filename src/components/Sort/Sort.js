import React from 'react';

export const Sort =() => {
  return (
    <div>
      <p> This is the sort page </p>
    </div>
  );
};