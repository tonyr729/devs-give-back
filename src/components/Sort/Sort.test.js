import { shallow } from 'enzyme';
import React from 'react';
import { Sort } from './Sort'

describe('Sort', () => {
  it('should match snapshot', () => {
    const sort = shallow(<Sort  />)

    expect(sort).toMatchSnapshot();
  });
});