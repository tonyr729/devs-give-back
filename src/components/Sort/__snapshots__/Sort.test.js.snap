// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sort should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p>
         This is the sort page 
</p>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " This is the sort page ",
      },
      "ref": null,
      "rendered": " This is the sort page ",
      "type": "p",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
           This is the sort page 
</p>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " This is the sort page ",
        },
        "ref": null,
        "rendered": " This is the sort page ",
        "type": "p",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
